frontend/views/containers/modals/InviteByLink.vue

```diff
@@ -6,11 +6,8 @@ modal-template(ref='modal')
   .c-container
     i18n.is-title-4(tag='h3') Share this link to grant access to your group.
     i18n.has-text-1(tag='p') After the onboarding period has ended, everyone will be asked to vote on whether or not a new member should be added. But for now, enjoy 60 free passes!
-    .c-link
-      a.link.has-icon(:href='link' target='_blank' data-test='invitationLink')
-        i.icon-link
-        | {{link}}
-    i18n.has-text-1(tag='p' :args='{ expireDate }') This invite link expires on the {expireDate}.
+    link-to-copy.c-link(:link='link')
+    i18n.has-text-1(tag='p' :args='{ expireDate }') This invite link expires on { expireDate }.
```

Comment:
<img width="302" alt="Screen Shot 2020-01-23 at 5 31 14 PM" src="https://user-images.githubusercontent.com/138706/73038191-5c732f80-3e06-11ea-8a0e-5d6317c94529.png">

our string replacement function that i18n uses isn't very smart, and needs no spaces around the variable name

Reply:
fixed.